eve/main.yml

```diff
@@ -5,6 +5,205 @@ branches:
   user/*, feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*, dependabot/*:
     stage: pre-merge
 
+models:
+  # --- Re-usable steps (no parameters) ---
+  - Git: &git_pull
+      name: git pull
+      repourl: "%(prop:git_reference)s"
+      method: clobber
+      retryFetch: true
+      haltOnFailure: true
+  - ShellCommand: &setup_cache
+      name: Setup proxy cache
+      command: >
+          curl -s http://proxy-cache/setup.sh | sudo sh &&
+          . /usr/local/bin/use_scality_proxy_cache
+      haltOnFailure: true
+  - ShellCommand: &wait_for_docker
+      name: Wait for Docker daemon to be ready
+      command: |
+        bash -c '
+        for i in {1..150}
+        do
+          docker info &> /dev/null && exit
+          sleep 2
+        done
+        echo "Could not reach Docker daemon from buildbot worker" >&2
+        exit 1'
+      haltOnFailure: true
+  - ShellCommand: &build_all
+      name: Build everything
+      env:
+        PYTHON_SYS: python3.6
+      # There are 3 CPUs available for Docker, and 1 for `doit` in the Pod.
+      # Given the network IO-bound nature of some of the build steps, and
+      # most build steps running in Docker, set concurrency to 4.
+      command: ./doit.sh -n 4
+      usePTY: true
+      haltOnFailure: true
+  - ShellCommand: &ssh_ip_setup
+      name: Report IP and install SSH keys
+      command: >
+        ip a &&
+        mkdir -p ~/.ssh &&
+        echo "%(secret:ssh_pub_keys)s" >> ~/.ssh/authorized_keys
+  - ShellCommand: &check_iso_checksum
+      name: Check image with checksum
+      command: sha256sum -c SHA256SUM
+      haltOnFailure: true
+  - Upload: &upload_artifacts
+      name: Upload artifacts
+      source: artifacts
+  - Upload: &upload_report_artifacts
+      name: Upload sosreport logs to artifacts
+      source: sosreport
+  - ShellCommand: &collect_sosreport
+      name: Collect logs using sosreport
+      command: >
+        sudo sosreport --all-logs
+        -o metalk8s -kmetalk8s.podlogs=True
+        -o containerd -kcontainerd.all=True -kcontainerd.logs=True
+        --batch --tmp-dir /var/tmp &&
+        sudo chown eve:eve /var/tmp/sosreport*
+  - ShellCommand: &wait_debug
+      name: Debug step - wait befor allowing resource destruction
+      timeout: 3600
+      command: sleep 3600
+      doStepIf: false
```

Comment:
I think we should use some property defaulted to False for this doStepIf, so we would be able to trigger debug on build without changing this file and doing a debug commit.